CMakeLists.txt

```diff
@@ -2,21 +2,23 @@ cmake_minimum_required(VERSION 3.8)
 project(snmalloc C CXX)
 
 include(CheckCXXCompilerFlag)
-
+include(CheckIncludeFile)
 option(USE_SNMALLOC_STATS "Track allocation stats" OFF)
 option(SNMALLOC_CI_BUILD "Disable features not sensible for CI" OFF)
 option(USE_MEASURE "Measure performance with histograms" OFF)
 option(EXPOSE_EXTERNAL_PAGEMAP "Expose the global pagemap" OFF)
 option(EXPOSE_EXTERNAL_RESERVE "Expose an interface to reserve memory using the default memory provider" OFF)
 option(SNMALLOC_RUST_SUPPORT "Build static library for rust" OFF)
 option(SNMALLOC_QEMU_WORKAROUND "Disable using madvise(DONT_NEED) to zero memory on Linux" Off)
-option(SNMALLOC_ANDROID "Build Target Android" OFF)
+option(SNMALLOC_ANDROID "Set cross build target to Android" OFF)
 set(CACHE_FRIENDLY_OFFSET OFF CACHE STRING "Base offset to place linked-list nodes.")
 
 if (SNMALLOC_ANDROID)
-set(CMAKE_SYSTEM_PROCESSOR "arm")
+  set(CMAKE_SYSTEM_PROCESSOR "arm")
 endif()
 
+check_include_file(execinfo.h HAS_EXECINFO_H)
```

Comment:
This macro can help us check the header file directly.

Reply:
Do we need this as a configure-time check?  C++17 standardised `__has_include()` and we depend on a C++17 compiler.